locale/templates/LC_MESSAGES/django.pot

```diff
@@ -4091,6 +4083,27 @@ msgstr ""
 msgid "More information on Developer Accounts"
 msgstr ""
 
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:3
+msgid "Firefox Android compatibility"
+msgstr ""
+
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:4
+msgid ""
+"Did you test your extension to ensure it works as expected on Firefox "
+"Android? If not, please see <a target=\"_blank\" rel=\"noopener noreferrer\" "
+"href=\"https://extensionworkshop.com/documentation/develop/developing-"
+"extensions-for-firefox-for-android/\">Developing extensions for Firefox "
+"Android</a> for test guidance before submitting."
+msgstr ""
+
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:6
+msgid "Yes, I’ve tested my Firefox Android extension"
+msgstr ""
+
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:7
+msgid "No, I have not tested for Android compatibility"
```

Comment:
Why did we switch to "Android" directly, when we're actually talking about compatibility with Firefox for Android?